amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -8,492 +8,967 @@ input AMPLIFY {
+  This field is referenced by Domain id, so has to be in the same format: colonyAddress_nativeId
+  """
+  id: ID!
+  """Name of the Domain."""
   name: String!
+  """Description of the Domain."""
   description: String!
+  """Color associated with the Domain."""
   color: DomainColor!
+  """List of Domain metadata changelog entries."""
   changelog: [DomainMetadataChangelog!]
 }
 
+"""
+Represents a changelog entry for Domain metadata.
+"""
 type DomainMetadataChangelog {
+  """Transaction hash associated with the changelog entry."""
   transactionHash: String!
+  """Name of the Domain before the change."""
   oldName: String!
+  """Name of the Domain after the change."""
   newName: String!
+  """Color of the Domain before the change."""
   oldColor: DomainColor!
+  """Color of the Domain after the change."""
   newColor: DomainColor!
+  """Description of the Domain before the change."""
   oldDescription: String!
+  """Description of the Domain after the change."""
   newDescription: String!
 }
 
+"""
+Represents a Colony Funds Claim for all ERC20 tokens (except native chain tokens)
+"""
 type ColonyFundsClaim @model {
+  """Unique identifier for the Colony Funds Claim."""
   id: ID! @index(sortKeyFields: ["createdAt"])
+  """Token associated with the Colony Funds Claim."""
   token: Token! @hasOne
+  """Block number when the Funds Claim was created."""
   createdAtBlock: Int!
+  """Timestamp when the Funds Claim was created."""
   createdAt: AWSDateTime!
+  """Amount claimed in the Colony Funds Claim."""
   amount: String!
 }
 
-# This is really not elegant, but there's no other proper solution, as you can't
-# return the Token @hadOne relationship from the lambda function
-# Note that we don't need token details for the values that we're fetching (since
-# it's implied), but it would have been nice not to have to basically repeat the type
+"""
+Represents a native Colony Chain Funds Claim
+
+E.g., Token 0x0000...0000: ETH, xDAI, etc.
+"""
 type ColonyChainFundsClaim {
+  """Unique identifier for the Colony Chain Funds Claim."""
   id: ID!
+  """Block number when the Chain Funds Claim was created."""
   createdAtBlock: Int!
+  """Timestamp when the Chain Funds Claim was created."""
   createdAt: AWSDateTime!
+  """Timestamp when the Chain Funds Claim was last updated."""
   updatedAt: AWSDateTime!
+  """Amount claimed in the Colony Chain Funds Claim."""
   amount: String!
 }
 
+"""
+Represents a Colony balance for a specific domain and token.
+"""
 # This is not a @model since it will only be returned by a lambda function
 # so don't need to create tables for them
 # Note that we also need input types since it treats the `balances` as available
 # to "write" to at creation time, even though, even if you were to write to
 # you'd never actually be able to read that data since the return is overwritten
 # by the lambda function
 type ColonyBalance {
+  """**Deprecated** Unique identifier for the Colony Balance."""
```

Comment:
How come id is marked as deprecated?

Reply:
@chmanie Sorry I might have misled you, even though this isn't a model Raul seems to be using this field (although it's only ever set and never read, so maybe it could get removed in a separate issue).